backend/models/User.js

```diff
@@ -2,32 +2,28 @@ import mongoose from 'mongoose';
 
 const UserSchema = mongoose.Schema(
     {
-        firstName: {
-            type: String, 
-            required: true
-        },
-        lastName: {
-            type: String, 
-            required: true
-        },
-        age: {
-            type: Number, 
-            required: true
-        }, 
-        username: {
-            type: String,
-            required: true,
-            unique: true
-        },
-        password: {
-            type: String,
-            required: true
-        },
-        description: {
-            type: String,
-            required: true
-        },
```

Comment:
when you get to sign in with google, you will notice that you will need to store something called a `sub` which is the token for that user. That may be their "password" in a sense, but i dont particularly like that.